utilities/transactions/lock/range/range_tree/range_tree_lock_manager.cc

```diff
@@ -329,6 +329,10 @@ RangeLockManagerHandle::Counters RangeTreeLockManager::GetStatus() {
       res.escalation_count = status->value.num;
       continue;
     }
+    if (strcmp(status->keyname, "LTM_WAIT_COUNT") == 0) {
```

Comment:
I know this may be beyond the scope of this PR, but please use constants or typed enums for the key names.

Reply:
Beyond the scope of this PR, so I am ok with current impl